osbs/conf.py

```diff
@@ -266,12 +266,23 @@ def get_registry_secrets(self):
         else:
             return []
 
-    def get_registry_api_versions(self):
+    def get_registry_api_versions(self, platform=None):
         value = self._get_value("registry_api_versions",
                                 self.conf_section,
                                 "registry_api_versions",
                                 default='v1,v2')
-        return value.split(',')
+        if not platform:
+            return value.split(',')
+        else:
+            section = 'platform:{0}'.format(platform)
+            enable_v1 = self._get_value("enable_v1", section, "enable_v1", False, True)
+            if enable_v1:
+                return value.split(',')
+            else:
+                if 'v2' in value:
```

Comment:
How about splitting the value right at the start? `versions = value.split(',')`

That would save repeating it, and make this line 'if 'v2' in versions:`, i.e. testing to see whether "v2" is one of the comma-separated values, rather than testing that it's a substring of the unsplit string (which would also match "v20,v21").